### DISCUSSION TITLE
DC intro will open everytime, instead of first

### DISCUSSION THREAD

**Discussion by Tenexxt** [ppy/osu#29451] [Q&A] [answered]
### Type

Game behaviour

### Bug description

Im not 100% sure if this is a bug, but it would make more sense for the intro to apear only for the first time, and unless the game is restarted for the cutscene to be skipped.

### Screenshots or videos

https://github.com/user-attachments/assets/cee2f188-44a5-4295-ac55-ab948c91dcc3

### Version

osu! version: Lazer 2024.816.0 (latest)

### Logs

[compressed-logs.zip](https://github.com/user-attachments/files/16637688/compressed-logs.zip)

**Comment by Jinkku** [comment, marked as answer]
That is intentional. Some players want to see the intro every time they open Daily Challenge
